lightning/src/onion_message/messenger.rs

```diff
@@ -828,6 +874,27 @@ where
 		)
 	}
 
+	/// Forwards an [`OnionMessage`] to `peer_node_id`. Useful if we initialized
+	/// the [`OnionMessenger`] with [`Self::new_with_offline_peer_interception`]
+	/// and want to forward a previously intercepted onion message to a peer that
+	/// has just come online.
+	pub fn forward_onion_message(
```

Comment:
Could you move this somewhere below the sending helper methods? Otherwise, it breaks the reading flow of `send_onion_message`.